netdcli/netdcli.go

```diff
@@ -182,10 +186,18 @@ func init() {
 		"cfg",
 		false,
 		"JSON file describing the global and network intent. Use '-' to read configuration from stdin")
+	flagSet.StringVar(&opts.stateStore,
+		"state-store",
+		utils.EtcdNameStr,
+		"State store to use")
 	flagSet.StringVar(&opts.etcdURL,
 		"etcd-url",
 		"http://127.0.0.1:4001",
 		"Etcd cluster url")
+	flagSet.StringVar(&opts.consulURL,
+		"consul-url",
+		"http://127.0.0.1:8500",
+		"Consul cluster url")
 	flagSet.StringVar(&opts.tenant,
```

Comment:
since we know these both take urls, how about `-state-store -connect` instead of the multiple parameters, and then we can sort out the logic for handling connect at `executeOpts` time.

WDYT?